From: Shengjing Zhu <zhsj@debian.org>
Date: Sat, 23 Jan 2021 00:42:41 +0800
Subject: disable runhcs option in cri config

---
 pkg/cri/server/helpers.go | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/pkg/cri/server/helpers.go b/pkg/cri/server/helpers.go
index 3514e68..5255bbb 100644
--- a/pkg/cri/server/helpers.go
+++ b/pkg/cri/server/helpers.go
@@ -24,7 +24,6 @@ import (
 	"strings"
 
 	"github.com/BurntSushi/toml"
-	runhcsoptions "github.com/Microsoft/hcsshim/cmd/containerd-shim-runhcs-v1/options"
 	"github.com/containerd/containerd"
 	"github.com/containerd/containerd/containers"
 	"github.com/containerd/containerd/plugin"
@@ -325,8 +324,6 @@ func getRuntimeOptionsType(t string) interface{} {
 		return &runcoptions.Options{}
 	case plugin.RuntimeLinuxV1:
 		return &runctypes.RuncOptions{}
-	case runtimeRunhcsV1:
-		return &runhcsoptions.Options{}
 	default:
 		return &runtimeoptions.Options{}
 	}
